src/main/java/edu/ie3/datamodel/io/csv/BufferedCsvWriter.java

```diff
@@ -79,12 +106,12 @@ protected final void writeFileHeader(String[] headLineElements) throws IOExcepti
    * @param entries Entries to write to the line of the file
    * @throws IOException If writing is not possible
    */
-  protected final void writeOneLine(String[] entries) throws IOException {
+  private final void writeOneLine(String[] entries) throws IOException {
```

Comment:
either make this method `protected` again or remove the `final` keyword as private methods are always implicitly `final`